app/src/app/framework/object/objects-by-type/objects-by-type.component.html

```diff
@@ -16,6 +17,8 @@ <h1><i class="{{type.render_meta?.icon}}"></i> {{type.label}}</h1>
             (searchChange)="onSearchChange($event)">
 </cmdb-table>
 
+{{selectedObjects | json}}
+
```

Comment:
Think you have forgotten something here ;-)

Reply:
Oops, good catch.